themes/survey/vanilla/views/subviews/privacy/privacy_text.twig

```diff
@@ -14,17 +14,48 @@
     Show the privacy message and the data security message will be available in a modal
 
 #}
-
-<!-- Privacy message -->
-<br/>
-<div class="row">
-    <div class="limit-text-window {{ aSurveyInfo.class.privacydatasectextbody }}">
-        {{ aSurveyInfo.datasecurity_notice }}
+<div class="privacy-block">
+    {% set describeprivacy = "" %}
+    {% if aSurveyInfo.datasecurity_notice %}
+        <div class="{{ aSurveyInfo.class.privacydatasectextbody }}" id="datasecurity_notice">
```

Comment:
This should get back its "limit-text-window" class. I think the "{{ aSurveyInfo.class.privacydatasectextbody }}" is fine where it is now, so I wouldn't change it.

Reply:
`<div class="limit-text-window {{ aSurveyInfo.class.privacydatasectextbody }}" id="datasecurity_notice">` ?

limit-text-window class didn't exist in css : right ?

Reply:
yeah looks okay, it does though themes/survey/vanilla/css/theme.css:767

Reply:
https://master.sondages.pro/256771?newtest=Y&lang=en didn't undestand readson of the border … but …

Reply:
yeah not the most optimal display but i don't want to change the design that got agreed on at some point

Reply:
Yes :) totally right

Reply:
I compare with previous to see if checkbox is inside the border.

Reply:
yeah checkbox was outside the border, just one last thing before merging could you add a padding to the "privacy" class of 20px before it had a "<br>" as first child, that gave it padding.

Reply:
Ok : same in old. i remove a br.